background_scripts/bg_utils.js

```diff
@@ -14,6 +14,41 @@ const BgUtils = {
       : false;
   },
 
+  // Get a query dictionary for `chrome.tabs.query` that will only return the visible tabs.
+  visibleTabs() {
```

Comment:
Since this object isn't yet used anywhere other than background_scripts/main.js, I think it should live in there. We can declare it when the file is initialized:

```javascript
const visibleTabsQueryArgs = { currentWindow: true };
if (BgUtils.isFirefox()) {
  // Only Firefox supports hidden tabs.
  visibleTabsQueryArgs.hidden = false;
}
```